README.md

```diff
@@ -2,42 +2,44 @@
+There are two ways to access [Google Cloud Storage](https://cloud.google.com/storage/). The old S3 API and the new JSON API. `Fog::Storage::Google` will automatically direct you to the appropriate API based on the credentials you provide it.
+
+ * The [XML API](https://developers.google.com/storage/docs/xml-api-overview) is almost identical to S3. Use [Google's interoperability keys](https://cloud.google.com/storage/docs/migrating#keys) to access it.
+ * The new [JSON API](https://developers.google.com/storage/docs/json_api/) is faster and uses auth similarly to the rest of the Google Cloud APIs using a [service account private key](https://developers.google.com/identity/protocols/OAuth2ServiceAccount).
 
 ## Compute
 
 Google Compute Engine is a Virtual Machine hosting service. Currently it is built on version [v1](https://developers.google.com/compute/docs/reference/v1/) of the GCE API.
 
-Our implementation of the API currently supports
+As of 2015-12-07, we believe Fog for Google Compute engine (`Fog::Compute::Google`) is feature complete.
 
- * Server creation, deletion and bootstrapping
- * Persistent Disk creation and deletion
- * Image lookup
- * Network and Firewall configuration
- * Operations
- * Snapshots
- * Instance Metadata
- * Project Metadata
+If you are using Fog to interact with GCE, please keep Fog up to date and [file issues](https://github.com/fog/fog-google/issues) for any anomalies you see or features you would like.
 
-Features we are looking forward to implementing in the future:
+## SQL
 
- * Image creation
- * Load balancer configuration
+Fog implements [v1beta3](https://cloud.google.com/sql/docs/admin-api/v1beta3/) of the Google Cloud SQL Admin API. This is a currently deprecated API. Pull Requests for updates would be greatly appreciated.
 
-If you are using Fog to interact with GCE, please keep Fog up to date and [file issues](https://github.com/fog/fog-google/issues) for any anomalies you see or features you would like.
+## DNS
+
+Fog implements [v1](https://cloud.google.com/dns/api/v1/) of the Google Cloud DNS API. We are always looking for people to improve our code and test coverage, so please [file issues](https://github.com/fog/fog-google/issues) for any anomalies you see or features you would like.
+
+## Monitoring
+
+Fog implements [v2beta1](https://cloud.google.com/monitoring/v2beta2/) of the Google Cloud Monitoring API. This is a currently deprecated version of the API. Pull requests for updates would be greatly appreciated.
 
 ## Installation
 
-Add this line to your application's Gemfile:
+Add the following two lines to your application's `Gemfile`:
 
 ```ruby
 gem 'fog-google'
+gem 'google-api-client'
```

Comment:
I would set it to something like this, otherwise we may get issues with people using either a very old untested client or the incompatible 0.9 version:
gem 'google-api-client', '< 0.9', '>= 0.6.2'